### PR TITLE
Add Passport and Azure AD Oauth2 to project dependencies

### PR DESCRIPTION
**Pull request by LucasCairns**

### CODE REVIEW

File: server/routes/auth.js [hunk 1]
```diff
@@ -1,73 +1,13 @@
 const express = require('express');
-const { path, pathOr } = require('ramda');
 
-const createAuthRouter = ({
-  logger,
-  formParser,
-  authenticateUser,
-  createUserSession,
-}) => {
+const createAuthRouter = ({ signIn, signInCallback, signOut }) => {
```
Comment (psoleckimoj) [inline server/routes/auth.js:3]: Only thought is logging since the logger is passed in on creation. But we can do that in a later PR

**Review by psoleckimoj — DISMISSED** [review]
LGTM

File: server/routes/auth.js (hunk 1 above)
Comment (LucasCairns) [reply server/routes/auth.js:3]: We have a ticket on the sprint backlog for that IIRC

**Review by psoleckimoj — APPROVED** [review]
LGTM again

**Review by davidthomas22 — COMMENTED** [review]
LGTM

**Pull request merged**